week2/css/style.css

```
@@ -0,0 +1,281 @@
+body {
+  background-color: #eaefff;
+}
+
+/*헤더*/
+header {
+  display: grid; /*표 처럼 나누고 그 안에 요소들을 넣는 것*/
+  grid-template-columns: 1fr 1fr 1fr; /*그리드 칸을 나누는 거임, 나는 같은 크기 3개로 나누고 싶으니까 1fr 3개*/
+  align-items: center;
+  background-color: #bad2ff;
+  padding: 1.2rem;
+  min-width: 750px;
+}
+
+.leftHeader {
+  color: white;
+  text-align: left;
+}
+
+.centerHeader {
+  color: white;
+  text-align: center;
+  font-size: 1.2rem;
+  font-weight: bold;
+}
+
+.rightHeader {
+  color: white;
+  text-align: right;
+  font-size: 0.9rem;
+}
+
+/*필터*/
+.filter {
+  margin: 2rem auto;
+  background-color: #bad2ff;
+  border-radius: 4px;
+  width: 60%;
+  min-width: 750px; /*최소로 작아질 수 있는 값을 지정해두니까 그 이상 작아졌을 때 스크롤 하도록 나와서 인풋 값이 넘치는 일이 발생하지 않음, 이 방법 말고 다른 방법은 없을까?*/
+}
+
+.filter-input {
+  display: grid;
+  grid-template-columns: repeat(6/1fr);
+  padding: 1rem;
+  gap: 0.5rem;
+}
+
+.filter-name,
+.filter-englishName {
+  grid-column: 1/7;
+}
+
+.filter-github {
+  grid-column: 1/3;
+}
+
+.filter-gender {
+  grid-column: 3/5;
+}
+
+.filter-role {
+  grid-column: 5/7;
+}
+
+.filter-firstWeekGroup {
+  grid-column: 1/4;
+}
+
+.filter-secondWeekGroup {
+  grid-column: 4/7;
+}
+
+.filter-name,
+.filter-englishName,
+.filter-github,
+.filter-gender,
+.filter-role,
+.filter-firstWeekGroup,
+.filter-secondWeekGroup {
+  display: flex;
+}
+
+.filter label {
+  background-color: #e9e9e9;
+  min-width: 8rem;
+  height: 3.5rem;
+  text-align: center; /*세로 중앙 정렬*/
+  align-content: center; /*가로 중앙 정렬*/
+  color: #51505e;
+  font-size: 0.9rem;
+  font-weight: 700;
+}
+
+.filter input {
+  width: calc(100% - 8rem); /* 수식 띄어쓰기 안 하면 안 먹힘!!*/
+  border: none; /*검정 테두리 없애기*/
+  padding: 0.5rem;
+}
+
+.filter select {
+  width: 100%;
+  border: none;
+  padding: 0.5rem;
+}
```

```suggestion
.filter input,
.filter select {
  width: 100%
  border: none;
  padding: 0.5rem;
}
```
저는 두 스타일이 똑같아도 되지 않을까 생각하는데 input의 width에만 따로 계산값을 넣어주신 이유가 있는지 궁금합니다!

그리고 border: none과 같이 기본 세팅값을 수정하는건 reset.css 파일에서 한번에 관리하면 더 좋을 것 같아요!